tools/msstitch/datatypes_conf.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0"?>
+<datatypes>
+  <registration>
+    <datatype extension="kr" type="galaxy.datatypes.tabular:Tabular" subclass="true" />
```

Comment:
Can we get this into Galaxy?

Reply:
Thanks @bgruening ! I have to read up a bit about the suite file and also fix the error detection thing, but here are some indent fixes. Cant believe how much my `filetype plugin indent on` in `.vimrc` would mess things up.